src/Message/Message.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from '../Icon'
+import styles from './Message.module.scss'
+
+const Message = ({
+  wrapperClassName,
+  className,
+  variant,
+  children,
+  icon,
+  iconClassName,
+  ...props,
+}) => (
+  <div className={cx(wrapperClassName, styles.wrapper, styles[variant])} {...props}>
+    {icon && (
+      <Icon
+        fill="inherit"
+        type={icon}
+        className={cx(iconClassName, styles.icon, styles[`icon-${variant}`])}
+      />
+    )}
+    <div>
+      {children}
+    </div>
+  </div>
+);
+
+Message.propTypes = {
+  className: PropTypes.string,
+  iconClassName: PropTypes.string,
+  wrapperClassName: PropTypes.string,
+  variant: PropTypes.oneOf(['info', 'warn', 'success', 'error']),
+  icon: PropTypes.string,
+  children: PropTypes.any.isRequired,
+};
+
+Message.defaultProps = {
+  variant: 'info',
+  wrapperClassName: '',
+  className: '',
+  iconClassName: '',
```

Comment:
After fixing everything, remove unnecessary props definition in this component

Reply:
Done.